src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -177,7 +177,7 @@ def choose_standard_households(self, target_number_sims: int) -> pd.DataFrame:
         # get all simulants per household
         chosen_persons = pd.merge(
             chosen_households,
-            self.population_data["persons"],
+            self.population_data["persons"][metadata.PERSONS_COLUMNS_TO_INITIALIZE],
```

Comment:
You should do this subsetting in `_load_population_data` unless we need the other columns for some other reason.